<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWebEngineView" name="webEngineView">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>651</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 153, 204);
	background-color: white;
	border: 2px solid rgb(153, 153, 204);
	border-radius: 5px;</string>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>772</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:disabled {
background-color: gray;
}
QPushButton {
color: rgb(153, 153, 204);
background-color: white;
border: 2px solid rgb(153, 153, 204);
border-radius: 5px;
}


</string>
   </property>
   <property name="text">
    <string>입지 분석</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReqProperty">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 153, 204);
	background-color: white;
	border: 2px solid rgb(153, 153, 204);
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>매물 검색</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>자본금</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetPropertyList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>281</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(000, 000, 000);
	background-color: white;
	border: 2px solid rgb(153, 153, 204);
	border-radius: 5px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>매물 리스트</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelNoProperty">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>280</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string>매물이 없습니다.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>600</y>
     <width>293</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>매물 정보</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>600</y>
     <width>313</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>입지 분석 결과</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(000, 000, 000);
	background-color: white;
	border: 2px solid rgb(153, 153, 204);
	border-radius: 5px;</string>
   </property>
   <item>
    <property name="text">
     <string>1000만원 미만</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000만원 이상 2000만원 미만</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000만원 이상 3000만원 미만</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3000만원 이상 4000만원 미만</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4000만원 이상 5000만원 미만</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5000만원 이상 6000만원 미만</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6000만원 이상 7000만원 미만</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7000만원 이상 8000만원 미만</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8000만원 이상 9000만원 미만</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9000만원 이상 1억 미만</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1억 이상</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>620</y>
     <width>321</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_2
{
	border: 2px solid rgb(153, 153, 204);
}</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(000, 000, 255);</string>
    </property>
    <property name="text">
     <string>상승요인</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPopulationCount">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>177</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>거주인구수</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelExpectedSales">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>155</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>예상 매출</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(204, 000, 000);</string>
    </property>
    <property name="text">
     <string>하락요인</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetPositiveElements">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>30</y>
      <width>144</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetNegativeElements">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>30</y>
      <width>145</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelNoLocationAnalysis">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid black;
background-color: white;</string>
    </property>
    <property name="text">
     <string>매물을 선택하고 입지 분석을 눌러주세요</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetProperty" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>620</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widgetProperty{
color: rgb(153, 153, 204);
border: 2px solid rgb(153, 153, 204);
}</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>46</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>임대형태</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPropertyType">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>46</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>77</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>권리금</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPropertyDeposit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>주소</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPropertyEntitlement">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>77</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>주소</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>172</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>면적</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPropertyArea">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>172</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>주소</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>141</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>월세</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPropertyMonthly">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>141</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>주소</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>110</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>보증금</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>15</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>주소</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPropertyAddress">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>15</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>주소</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNoPropertyInformation">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid black;
background-color: white;</string>
    </property>
    <property name="text">
     <string>매물이 선택되지 않았습니다.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnQnA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 153, 204);
	background-color: white;
	border: 2px solid rgb(153, 153, 204);
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Q n A</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
